api/src/main/java/uk/gov/dhsc/htbhf/dwp/entity/Child.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.dhsc.htbhf.dwp.entity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDate;
+import java.util.UUID;
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.validation.constraints.Size;
+
+@Data
+@NoArgsConstructor
+@SuperBuilder
+@Entity
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
+@SuppressWarnings("PMD.AbstractClassWithoutAbstractMethod")
+public abstract class Child {
```

Comment:
As discussed, I think a Child interface with getter methods would be cleaner - one for a group discussion